multy_core/src/bitcoin/bitcoin_transaction.cpp

```diff
@@ -828,7 +828,21 @@ void BitcoinTransaction::update()
         }
     }
 
-    m_fee->validate_fee(calculate_diff(), get_transaction_serialized_size(WITH_POSITIVE_CHANGE_AMOUNT));
+    m_fee->validate_fee(calculate_diff(), get_transaction_serialized_size(WITH_POSITIVE_CHANGE_AMOUNT));   
+
+    size_t count_nonzero_destination = 0;
+    for (const auto& d : m_destinations)
+    {
+        if (d->amount.get_value() != 0)
+        {
+            ++count_nonzero_destination;
+        }
+    }
+    if (count_nonzero_destination == 0)
+    {
+        THROW_EXCEPTION2(ERROR_TRANSACTION_CHANGE_IS_TOO_SMALL_AND_NO_OTHER_DESTINATIONS,
+                "Transaction change is to small because higt fee.");
```

Comment:
"Transaction change is to small because higt fee" => "Transaction change is to small because of high fee."